daprovider/writer.go

```diff
@@ -1,18 +0,0 @@
-// Copyright 2021-2025, Offchain Labs, Inc.
-// For license information, see https://github.com/OffchainLabs/nitro/blob/master/LICENSE.md
-
-package daprovider
-
-import (
-	"context"
-)
-
-type Writer interface {
-	// Store posts the batch data to the invoking DA provider
-	// And returns sequencerMsg which is later used to retrieve the batch data
-	Store(
-		ctx context.Context,
-		message []byte,
-		timeout uint64,
-	) ([]byte, error)
-}
```

Comment:
This is backwards, we definitely want to keep the daprovider.Writer and we don't want to use the AnyTrust specific interface in its place. I'd avoid trying to unify these interfaces at this stage.